### PR TITLE
[Components] convertkit #8620

### PR DESCRIPTION
**Pull request by lcaresia**
### <samp>🤖 Generated by Copilot at 89ee5f4</samp>

Added support for tagging subscribers in ConvertKit. Updated the versions of the `convertkit` component, its actions, sources, and `package.json` file to reflect the new feature. Added `components/convenia` as a dependency in `pnpm-lock.yaml`.

### <samp>🤖 Generated by Copilot at 89ee5f4</samp>

> _We are the ConvertKit rebels_
> _We tag our subscribers with fire_
> _We use the `addTagToSubscriber` method_
> _To unleash our marketing desire_

### <samp>🤖 Generated by Copilot at 89ee5f4</samp>

*  Added the `tag` prop and the `addTagToSubscriber` method to the `convertkit.app.mjs` file to enable tagging subscribers in ConvertKit ([link](https://github.com/PipedreamHQ/pipedream/pull/8697/files?diff=unified&w=0#diff-0dd62370282695ace5e45f4abfb9ec7f33431bf8d001642b06219294b9d8d1f2R37-R48), [link](https://github.com/PipedreamHQ/pipedream/pull/8697/files?diff=unified&w=0#diff-0dd62370282695ace5e45f4abfb9ec7f33431bf8d001642b06219294b9d8d1f2R135-R142))
* Updated the version of the `convertkit` component in the `package.json` file to reflect the new feature ([link](https://github.com/PipedreamHQ/pipedream/pull/8697/files?diff=unified&w=0#diff-aa3a7fc63a2bd6eda358baa2399c6d51eca602d6716085e10135086d5cfa5144L3-R3))
* Updated the versions of the actions and sources that depend on the `convertkit.app.mjs` file to match its version ([link](https://github.com/PipedreamHQ/pipedream/pull/8697/files?diff=unified&w=0#diff-cadfced3d2aa4d857cbeb7b7d91ce9415aced8eadc5bfb443c62d9dbee33132dL7-R7), [link](https://github.com/PipedreamHQ/pipedream/pull/8697/files?diff=unified&w=0#diff-07731e4ba18488c65331edbfb66dced5feb43b67d9f706180cd11f73e90879a5L8-R8), [link](https://github.com/PipedreamHQ/pipedream/pull/8697/files?diff=unified&w=0#diff-c721e05a0140f5448c93b26940f33e760d6d4890073b9e4c5c211f0b4b4eeffeL7-R7), [link](https://github.com/PipedreamHQ/pipedream/pull/8697/files?diff=unified&w=0#diff-65e8603cefa77add61811e77b4fcf5494dd2368cde228cbee8d9b776e23e76e6L9-R9), [link](https://github.com/PipedreamHQ/pipedream/pull/8697/files?diff=unified&w=0#diff-cbc110e484544e37e3f014fef14db7ddc8c42f5b7049fedb4702d1b7636a7625L9-R9), [link](https://github.com/PipedreamHQ/pipedream/pull/8697/files?diff=unified&w=0#diff-975f865f46e0e97a936e96d3ec3fe8dac2b387012b6fa02c26dbd97bead16d23L10-R10))
* Added the `components/convenia` entry to the `pnpm-lock.yaml` file as a dependency of the `convertkit` component ([link](https://github.com/PipedreamHQ/pipedream/pull/8697/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR1128-R1130))

### CODE REVIEW

**Comment by lcaresia** [comment]
image

File: components/convertkit/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/bitbucket",
```
Comment (GTFalcao) [inline components/convertkit/package.json:2]: shouldn't this be `@pipedream/convertkit`?

File: components/convertkit/actions/add-tag-to-subscriber/add-tag-to-subscriber.mjs [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import convertkit from "../../convertkit.app.mjs";
+
+export default {
+  key: "convertkit-add-tag-to-subscriber",
+  name: "Add tag to a subscriber",
+  description: "Add subscriber to a form. [See docs here](https://developers.convertkit.com/#tag-a-subscriber)",
```
Comment (GTFalcao) [inline components/convertkit/actions/add-tag-to-subscriber/add-tag-to-subscriber.mjs:6]: I think the description needs to be updated

**Review by GTFalcao — CHANGES_REQUESTED** [review]
Moving forward since the requested changes do not impact functionality

**Pull request merged**